nemesis/changes.py

```diff
@@ -441,18 +438,19 @@ def to_remiem():
                 == True
             ):
                 checkpoint += 1
-                print("Checkpoint reached: ", checkpoint)
+                logger.debug("Checkpoint reached: ", checkpoint)
```

Comment:
Will crash, change to `logger.debug(f"Checkpoint reached: {checkpoint}")

Reply:
ah I missed one. Thanks for the catch.